### DISCUSSION TITLE
Documentation request: buffering

### DISCUSSION THREAD

**Discussion by 8fZJZYdV** [FrancescoCeruti/linux-show-player#307] [Q&A]
Could you please explain Media Settings -> URI Input -> Use Buffering feature? Been browsing through the Manual, but couldn't find it. 

Uncertain if it is the right place to request this. Beg pardon if it isn't.

**Comment by FrancescoCeruti** [comment]
In theory it could be used if you need to stream a live source, in practice I've never tested it properly

> Uncertain if it is the right place to request this. Beg pardon if it isn't.

It usually best to use the "discussions" section for question (I'll move the issue there)
